ui/app/AppLayouts/Chat/ChatColumn/Message.qml

```diff
@@ -56,7 +56,7 @@ Item {
         }
     }
 
-    function clickMessage(isProfileClick, isSticker = false, isImage = false, image = null) {
+    function clickMessage(isProfileClick, isSticker = false, isImage = false, image = null, emojiOnly = false) {
```

Comment:
We should start exploring breaking the message context menu into multiple, composable declarative components, instead of having this one-size-fits-all solution that grows into a configuration mess